senders/victorops/send_test.go

```diff
@@ -0,0 +1,123 @@
+package victorops
+
+import (
+	"testing"
+	"time"
+
+	"github.com/moira-alert/moira/senders/victorops/api"
+
+	"github.com/moira-alert/moira"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+type MockImageStore struct {
```

Comment:
For some reason I did not immediately notice how you made this decision in pagerduty =(
For these stubs we use gomock. You can add your interface in the files generate_mocks.(cmd and sh) and execute then. Let's try to generate mock for image store and replace them.